ouroboros-network-framework/src/Ouroboros/Network/Snocket.hs

```diff
@@ -363,10 +363,10 @@ namedPipeSnocket ioManager path = Snocket {
                  (Win32.pIPE_ACCESS_DUPLEX .|. Win32.fILE_FLAG_OVERLAPPED)
                  (Win32.pIPE_TYPE_BYTE .|. Win32.pIPE_READMODE_BYTE)
                  Win32.pIPE_UNLIMITED_INSTANCES
-                 maxBound
-                 maxBound
-                 0
-                 Nothing
+                 65536   -- outbound pipe size
+                 16384   -- inbound pipe size
```

Comment:
[sduSize](https://github.com/input-output-hk/ouroboros-network/blob/master/network-mux/src/Network/Mux/Bearer/NamedPipe.hs#L40) for named pipes is `24576'

Reply:
Sure, but think about our protocols. For the local case there are no big messages from client -> node, just the chain sync intersection request. The reply size can be big. So 64k matches the pipe size on Linux, so we should get a similar amount of buffering & context switching. 16k on inbound is more than enough to queue lots of small request messages.

Reply:
You're right, we only need the `outbound` to be greater than `sduSize`.